tests/vibe.web.rest.error/source/app.d

```diff
@@ -0,0 +1,87 @@
+import std;
+import vibe.core.core;
+import vibe.core.stream;
+import vibe.data.bson;
+import vibe.data.json;
+import vibe.data.serialization;
+import vibe.http.client;
+import vibe.http.router;
+import vibe.http.server;
+import vibe.stream.memory;
+import vibe.stream.operations;
+import vibe.web.rest;
+
+void main()
+@safe {
+	auto settings = new HTTPServerSettings;
+	settings.port = 0;
+	settings.bindAddresses = ["127.0.0.1"];
+	auto router = new URLRouter;
+    router.registerRestInterface(new Server);
+    router.post("/api/error1", &handleError1);
+    router.post("/api/error2", &handleError2);
+	auto listener = listenHTTP(settings, router);
+	scope (exit) listener.stopListening();
+	immutable addr = listener.bindAddresses[0];
+
+	auto api = new RestInterfaceClient!API("http://"~addr.toString);
+
+    // Test that regular exception lead to JSON error
+    try {
+        api.getError();
+        assert(0);
+    } catch (RestException exc) {
+        assert(exc.message() == "Something very bad happened");
+        assert(exc.jsonResult["statusDebugMessage"].get!string.canFind("object.Exception@source/app.d"));
```

Comment:
Fails on Windows due to different directory separator: `/` vs `\`. On Windows it is `core.exception.AssertError@source\app.d(35): Assertion failure`